SQL/JSON_Parsing.js

```diff
@@ -0,0 +1,65 @@
+// Not completed list request
+if(req.body['all'])
+{
+	// Make the query string to pull back a single user query 
+	var queryString = 'SELECT US.CookieId, U.Name AS \'User\', US.SearchType, US.SearchDepth,'
+	queryString += ' URL.Url AS \'RootUrl\', URLS.Url AS \'SourceUrl\', URLD.Url AS \'DestinationUrl\',' 
+	queryString += ' URLS.Id AS \'SourceId\', URLD.Id AS \'DestinationId\''
+	queryString += ' FROM wc_UserSearches US'
+	queryString += ' INNER JOIN wc_SingleGraphs SG ON US.Id = SG.UserSearchId'
+	queryString += ' INNER JOIN wc_Edges E ON SG.Edge = E.Id'
+	queryString += ' INNER JOIN wc_Urls URL ON US.RootUrl = URL.Id'
+	queryString += ' INNER JOIN wc_Users U ON US.UserId = U.Id'
+	queryString += ' INNER JOIN wc_Urls URLS ON E.SorcUrlId = URLS.Id'
+	queryString += ' INNER JOIN wc_Urls URLD ON E.DestUrlId = URLD.Id'
+	queryString += ' WHERE US.CookieID = 2;'	
+	
+	// Call the select statement
+	pool.query(queryString, function(err, rows, fields)
+	{		
+		if(err)
+		{
+			res.type('text/plain');	
+			res.send(err);
+			return;
+		}
+		res.type('text/plain');	
+		res.send(QueryResultsToJson(JSON.stringify(rows)));
+	});
+}
+	
+	
+function CrawlResultsToJson(raw) 
```

Comment:
Typically javascript function names are camelCase, while class constructors are PascalCase. Javascript didn't have real classes until recently, so people used the casing to know when they should expect the function to return a class instance.

This is an internal project, so we can do what we like, but my functions will be camelCase.

Reply:
Thats cool - I dont use JS ever day to day. Exclusively .NET so I dont know about the format idiosyncrasies. We probably should have laid out some rules before we started. But I will follow the rules you speak of going forwar